saxon.py

```diff
@@ -0,0 +1,69 @@
+import subprocess
+import os
+
+SAXON_PATH = "./oer.exports/lib/saxon9he.jar"
+DELIMINATOR = "END_OF_XML_BLOCK"
+MATH2SVG_PATH = "./oer.exports/xslt2/math2svg-in-docbook.xsl"
+
+
+class Saxon:
+
+    def __init__(self, saxon_path=SAXON_PATH, math2svg_path=MATH2SVG_PATH):
+        math2svg_path = os.path.abspath(math2svg_path)
+        saxon_path = os.path.abspath(saxon_path)
+        if not (os.path.exists(saxon_path) and os.path.isfile(math2svg_path)):
+            raise IOError
+        self.compile_cmd = "javac -cp {} CNX.java".format(
```

Comment:
Requires that the file CNX.java file be located in the same directory as saxon9he.jar.  See https://github.com/Connexions/oer.exports/pull/1205